### ISSUE TITLE
Remove -j8 invocation of CMake generated makefile

### ISSUE THREAD

**Issue by Makman2** [Makman2/CE3D#172]
I discovered a new nice tool online: [koding.com](http://koding.com)
The thing is when building CE3D with the `-j8` tag the memory consumption is too high, so the compilation aborts (after a few minutes since this is a hard limit case). Even on my virtual machines I have no significant speedup (@sils1297: Okay, I know that doesn't apply on real machines). The virtual machine on koding had 1GB RAM.
The next thing is that the lines are printed ugly, especially when looking through tests it's confusing.
So I would remove `-j8`.

**Comment by sils1297** [comment]
You're suggesting using VMs is more common than using real machines.

**Comment by sils1297** [comment]
If we fix #170 and use meson we probably wouldn't need to care and meson would use the optimal count of processes. How about using `-j4` as a compromise?

**Comment by Makman2** [comment]
Especially for testing, I would say yes.

**Comment by sils1297** [comment]
When anyone wants to test CE3D he can spawn up a docker container, there's no need for such crude things like VM's if one is using an operation system that doesnt own the user.

**Comment by Makman2** [comment]
`-j4`... Not sure. Anyway I would test this online with koding if this also works. I just wanted to throw this issue here, not fixing it instantly.
Also in koding you own your machine.
And if I see it right is a docker container not the same as a VM?

**Comment by sils1297** [comment]
A docker container is not a VM. And thats good because it doesnt suffers from performance loss. However it is an encapsulated environment so perfect for a testing system.

I know koding. I meant windows when I said you shouldn't use a system that owns you.

**Comment by sils1297** [comment]
or tries to

**Comment by Makman2** [comment]
windows has it's advantages and disadvantages... the same applies for linux :P
And then docker is nice. But virtual machines are also an important technology and commonly used, so the minimal requirements of CE3D should fit the slowest supported systems (like VMs).
Is my opinion :O

**Comment by sils1297** [comment]
Yes but compiling CE3D is not the same as using CE3D. Compiling should be
possible but if memory goes up in that manner thats an indication that
something else is going very wrong IMO.

2015-02-21 16:58 GMT+01:00 Makman2 <notifications@github.com>:

**Comment by Makman2** [comment]
Yes sorry I meant requirements for compilation and not for using.
1GB is not that much, if you open 8 compiling threads (that do quite much stuff), together with the OS that can exceed the limit. But if that's just a really bad issue of koding then you're right.
But again the console output of the compilation bothers me xD

**Comment by sils1297** [comment]
The output is very well readable. However if you remove -j8 the usual
_developer_ (and noone develops in a VM if he's not forced to) will have to
wait some time for every compilation. I really hate waiting, you know that?

2015-02-21 17:04 GMT+01:00 Makman2 <notifications@github.com>:
